Protocol.md

```diff
@@ -300,6 +317,173 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+# Channel
+
+## Overview
+
+The lifetime of a channel is composed of its establishment, and the possibility for each party to offer the settlement or renewal of the contract within the channel, as well as to close the channel.
+
+The establishment protocol is depicted in the following [state machine](#state-machine-representation).
+
+<img src="./images/dlc_channel_establish_ta.svg">
+
+The protocol for renewal, settlement and closing of the channel is shown in the following [state machine](#state-machine-representation).
+
+<img src="./images/dlc_channel_settle_renew.svg">
+
+### Revocation
+
+Revocation is done following the mechanism introduced in [this paper](https://eprint.iacr.org/2020/476).
+
+### State machine representation
+ The state machine is represented as a timed automaton augmented with:
+* channel synchronization representing message passing between the processes (`?` indicating the receiving of a message while `!` indicates its sending),
+* edge guards only enabling state transition to be taken when the condition is satisfied (e.g. `Invalid` being true),
+* location invariants requiring the associated condition to hold for the automaton to stay in the location,
+* updates, setting the value of free variables and clocks. 
+
+Note that transitions setting the variable `Invalid` to true are omitted for clarity.
+
+## The `offer_channel` message
```

Comment:
The `CloseOffer` message seems to be missing.

Reply:
I think it makes sense to include a `feerate_per_vb` in the `CloseOffer` message.

Reply:
Thanks, added the message as well as the `feerate_per_vb`. In LN they actually use a range, which can be nice but also mean a more complex protocol (can't send the signature right away).